samples/scenarios/extract_only/sqlite_extract/dialect.xml

```diff
@@ -1,5 +1,1319 @@
+      <formula part='minute'>((CAST(%1 AS DATE) + EXTRACT(HOUR FROM %1) * INTERVAL &apos;1 HOUR&apos;) + EXTRACT(MINUTE FROM %1) * INTERVAL &apos;1 MINUTE&apos;)</formula>
+      <formula part='second'>(((CAST(%1 AS DATE) + EXTRACT(HOUR FROM %1) * INTERVAL &apos;1 HOUR&apos;) + EXTRACT(MINUTE FROM %1) * INTERVAL &apos;1 MINUTE&apos;) + EXTRACT(SECOND FROM %1) * INTERVAL &apos;1 SECOND&apos;)</formula>
+      <argument type='localstr' />
+      <argument type='datetime' />
+      <argument type='localstr' />
+    </date-function>
+  </function-map>
+  <supported-aggregations>
+    <aggregation value='AGG_SUM' />
+    <aggregation value='AGG_AVG' />
+    <aggregation value='AGG_MIN' />
+    <aggregation value='AGG_MAX' />
+    <aggregation value='AGG_STDEV' />
+    <aggregation value='AGG_STDEVP' />
+    <aggregation value='AGG_VAR' />
+    <aggregation value='AGG_VARP' />
+    <aggregation value='AGG_COUNT' />
+    <aggregation value='AGG_COUNTD' />
+    <aggregation value='AGG_YEAR' />
+    <aggregation value='AGG_QTR' />
+    <aggregation value='AGG_MONTH' />
+    <aggregation value='AGG_DAY' />
+    <aggregation value='AGG_HOUR' />
+    <aggregation value='AGG_MINUTE' />
+    <aggregation value='AGG_SECOND' />
+    <aggregation value='AGG_WEEK' />
+    <aggregation value='AGG_WEEKDAY' />
+    <aggregation value='AGG_MONTHYEAR' />
+    <aggregation value='AGG_MDY' />
+    <aggregation value='TRUNC_YEAR' />
+    <aggregation value='TRUNC_QTR' />
+    <aggregation value='TRUNC_MONTH' />
+    <aggregation value='TRUNC_WEEK' />
+    <aggregation value='TRUNC_DAY' />
+    <aggregation value='TRUNC_HOUR' />
+    <aggregation value='TRUNC_MINUTE' />
+    <aggregation value='TRUNC_SECOND' />
+  </supported-aggregations>
+  <sql-format>
+    <date-literal-escape value='Standard' />
+    <format-bool-as-value value='TrueFirst' />
+    <format-column-definition>
+      <local-type name='int'>
+        <remote-type name='I2' value='SMALLINT' />
+        <remote-type name='I4' value='INTEGER' />
+        <remote-type name='default' value='BIGINT' />
+      </local-type>
+      <local-type name='real'>
+        <remote-type name='R4' value='FLOAT(24)' />
+        <remote-type name='default' value='FLOAT(53)' />
+      </local-type>
+      <local-type name='str'>
+        <remote-type name='default' value='VARCHAR(%w)' />
+      </local-type>
+      <local-type name='datetime'>
+        <remote-type name='TIME' value='TIME' />
+        <remote-type name='DATE' value='DATE' />
+        <remote-type name='default' value='TIMESTAMP' />
+      </local-type>
+      <local-type name='date'>
+        <remote-type name='default' value='DATE' />
+      </local-type>
+      <local-type name='bool'>
+        <remote-type name='default' value='BOOLEAN' />
+      </local-type>
+      <local-type name='spatial'>
+        <remote-type name='default' value='GEOGRAPHY' />
+      </local-type>
+    </format-column-definition>
+    <format-create-table>
```

Comment:
Is there any way we can get to a more minimal representation of this dialect that still works (it could be slightly different).
I don't imagine for example format-create-table, format-drop-table, format-set-isolation-level, format-stored-proc-call matter in extract only context.
Others like format-order-by and format-null match the [default](https://github.com/tableau/connector-plugin-sdk/blob/dev-2020.3/samples/components/dialects/SQLDialect.tdd) (it looks like to me anyway), so I think those might be able to be removed as well.

Reply:
I removed all SQL formatting stuff that are same with SQLDialect in 4th and 5th commit, also including format-create-table, format-drop-table, format-set-isolation-level, format-stored-proc-call.